session_test.go

```diff
@@ -521,11 +540,11 @@ func (s *testSessionSuite) TestIssue827(c *C) {
 	r, err := GetRows(rs)
 	c.Assert(err, IsNil)
 	c.Assert(r, DeepEquals, expect)
-	currLastInsertID := se.LastInsertID()
+	currLastInsertID := se.GetSessionVars().PrevLastInsertID
```

Comment:
LastInsertID is reset to zero on every request, its value is set to PrevLastInsertID